src/flavors/pbdurham/config.json

```diff
@@ -731,8 +731,15 @@
             "name": "project_description",
             "optional": true
           },
+          {
+            "name": "delegates_header",
+            "restrictToGroup": "delegates",
```

Comment:
I think we will want this to be an array of groups, and call it `restrictToGroups`. For example, Durham's "internal staff committee" (or "tech reviewers") may want to be able to edit this field as well.